infrastructure/pcluster-manager.yaml

```diff
@@ -1,14 +1,31 @@
 Parameters:
   AdminUserEmail:
-    Description: Email address of administrative user setup by default.
+    Description: Email address of administrative user to setup by default (only with new Cognito instances).
     Type: String
-    MinLength: 1 # Force this to be selected
```

Comment:
question: is it possible to validate this constraint only when creating a new Cognito stack?

Reply:
Unfortunately I couldn't find anything like that, so I added the info in the description `(only with new Cognito instances)` to this field